charts/stable/statping/values.yaml

```diff
@@ -8,7 +8,9 @@
 image:
   repository: statping/statping
   pullPolicy: IfNotPresent
-  tag: "v0.90.65"
+  # -- image tag
+  # @default -- `.Chart.appVersion`
+  tag: ""
```

Comment:
Even though the common library will default to the appVersion, we still keep an explicit tag in our values.yaml, also so that it gets populated in the README.

Reply:
updated